frontend/tests/acceptance/api.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios");
+const { users } = require("./globals");
+
+module.exports = {
+  server: process.env.REACT_APP_SERVER_URL || "http://localhost:3000",
+  registerUser: async (name, email, password) => {
+    try {
+      await axios
+        .post(`${this.server}/auth/register`, { name: name, email: email, password: password })
+        .then(({ data }) => {
+          users.push({ email: email, password: password });
+          I.say(data.message);
+        })
+        .catch((err) => {
+          console.log(`Cannot register new user\n${err}`);
+        });
+    } catch (e) {
+      console.log(e);
+    }
+  },
+  deleteUser: () => {
+    if (users.length != 0) {
+      users.forEach(async (user) => {
+        try {
+          await axios
+            .delete(`${this.server}/auth/deleteuser`, { params: { email: user.email, password: user.password } })
+            .then(({ data }) => I.say(data.message))
```

Comment:
why using data as object??

Reply:
doing object destructing. So don't need to type `res.data.message` but simply `data.message`